packages/asc-ui/src/components/Menu/MenuToggle/MenuToggle.tsx

```diff
@@ -10,6 +10,7 @@ const MenuToggle: React.FC<Props> = ({
   onExpand,
   align = 'left',
   open,
+  title,
```

Comment:
Same here, title will be spread when passed, perhaps merge HTMLAttributes as props as well?